src/tester.rs

```diff
@@ -144,6 +150,28 @@ impl<'a> LangTester<'a> {
         self
     }
 
+    /// If set to `true`, this reads arguments from `std::env::args()` and interprets them in the
+    /// same way as normal cargo test files. For example if you have test "ab" and "cd" but only want
+    /// to run the latter:
+    ///
+    /// ```sh
+    /// $ <test bin> c
+    /// ```
+    ///
+    /// As this suggests, a simple substring search is used to decide which tests to run.
+    ///
+    /// You can get help on `lang_tester`'s options:
+    ///
+    /// ```sh
+    /// $ <test bin> --help
+    /// ```
+    ///
+    /// This option defaults to `true`.
+    pub fn use_cmdline_args(&'a mut self, use_cmdline_args: bool) -> &'a mut Self {
```

Comment:
Why might I want to want to switch this off? If I don't care to filter the tests, I won't run `test` with a filter. I'm guessing maybe in case there is ambiguity with other `test` command line arguments?

Reply:
Someone might run this as part of some non-cargo setup, and not want `lang_tester` to look at the command-line arguments. I don't have a concrete use-case for this.

Reply:
Fair enough